src/cutadapt/cli.py

```diff
@@ -296,6 +296,10 @@ def get_argument_parser() -> ArgumentParser:
         metavar="ERRORS",
         help="Discard reads whose expected number of errors (computed "
             "from quality values) exceeds ERRORS.")
+    group.add_argument("--max-average-error-rate", "--max-er", type=float, default=None,
```

Comment:
I’m worried that `--max-er` looks a bit too similar to `--max-ee` and may be confused with it. Maybe use `--max-aer` for the short option?